app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -153,8 +134,8 @@ docker tag kong:{{page.kong_versions[page.version-index].ce-version}}-alpine kon
  kong-ee</code></pre></div>
 {% endnavtab %}
 {% navtab Kong Gateway (OSS) %}
-<div class="copy-code-snippet"><pre><code>docker run -d --name kong \
- --network=kong-net \
+<div class="copy-code-snippet"><pre><code>docker run -d --name <div contenteditable="true">{CONTAINER_NAME}</div> \
+ --network=<div contenteditable="true">{NETWORK_NAME}</div> \
  -e "KONG_DATABASE=postgres" \
  -e "KONG_PG_HOST=kong-database" \
  -e "KONG_PG_USER=kong" \
```

Comment:
should these ones also be editable, as they were in the snippets above?

Reply:
added editable fields for these. Good catch 💯